test/PhylumExt.toml

```diff
@@ -0,0 +1,6 @@
+name = "test"
+description = "API injection test"
+entry_point = "main.ts"
+
+[permissions]
+net = ["http://blog.phylum.io/test/ing"]
```

Comment:
Was this just for your local testing? It's not used in any test and isn't in the normal location.

Reply:
Correct.